great_expectations/cli/suite.py

```diff
@@ -87,7 +87,17 @@ def suite_edit(suite, datasource, directory, jupyter, batch_kwargs):
     )
 
 
-def _suite_edit(suite, datasource, directory, jupyter, batch_kwargs, usage_event):
+def _suite_edit(
+    suite,
+    datasource,
+    directory,
+    jupyter,
+    batch_kwargs,
+    usage_event,
+    actually_send_usage_message=True,
```

Comment:
Could you rename this to `suppress_usage_message=False`? Minor, but I got thrown by "actually..." until I realized that it's because send_usage_message name is taken. :)